### ISSUE TITLE
Admin link on STS site not showing

### ISSUE THREAD

**Issue by johndcollins** [skoruba/Duende.IdentityServer.Admin#147]
Hi,
I'd like to get the admin link working on the STS site. It looks like it's supposed to be there:

<vc:identity-server-admin-link></vc:identity-server-admin-link>

However, it's not rendering on the site.
Is there a setting which controls this?

**Comment by skoruba** [comment]
Hi, do you have correct permissions to see this link? I mean admin permission.
thanks

**Comment by johndcollins** [comment]
Yes, I'm able to log in to the admin site with a direct link. Only have one admin user in the system.
